vta/python/vta/top/graphpack.py

```diff
@@ -246,32 +246,38 @@ def visit_call(self, call):
 
 class BT(Exception):
     pass
-def get_subgraph(expr, start_name, stop_name):
+def get_subgraph(expr, start_name, stop_name, start_name_idx=-1, stop_name_idx=-1):
```

Comment:
I suggest setting default `start_name_idx` being 0 (first occurrence of the operator), while keeping `stop_name_idx` being -1 (last occurrence).

Reply:
I feel we may need to set both value is -1 to make the logic be consistent, would keep this no change.

Reply:
I think there might be a misunderstanding.

Let's take a specific example. For a network with layers defined as
```
%0 = nn.conv2d
%1 = nn.batch_norm
%2 = nn.relu
%3 = nn.max_pool2d
%4 = nn.conv2d
%5 = nn.batch_norm
%6 = nn.relu
%7 = nn.max_pool2d
%8 = nn.conv2d
%9 = nn.global_avg_pool2d
%10 = nn.dense
%11 = nn.softmax
```
, we use your default configuration which means start_name="nn.max_pool2d", stop_name="nn.global_avg_pool2d", start_name_idx=-1, stop_name_idx=-1. What you would get would be the subgraph between %7 and %9. So, I'm suggesting setting `start_name_idx`=0, so that we could have the subgraph extracted between %3 and %9, which  I think would be more reasonable.